lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
This will break if we try to deploy it. Migration 0001 is already deployed and has been applied to stage/prod/edge, so we can't edit the migration file. We could squash 0002 and 0003 together if neither has been deployed.